src/classes/BDI_DataImport_TEST.cls

```diff
@@ -357,18 +357,29 @@ public with sharing class BDI_DataImport_TEST {
         ID ApexJobId = Database.executeBatch(bdi, 10);
         Test.stopTest();
     
+        Boolean usingAdv = ADV_PackageInfo_SVC.useAdv();
         // verify expected results
         List<Contact> listCon = getContacts();
-        System.assertEquals(1, listCon.size());
+        Integer numCons = usingAdv ? 2 : 1;
```

Comment:
This doesn't make sense to me as a desirable outcome. I believe that this test is verifying that if you import a contact that matches an existing contact, that person doesn't get a second contact record. Are you interpreting this differently?